### PR TITLE
PG-580 Release notes 2.0.0

### PR DESCRIPTION
**Pull request by nastena1606**

### CODE REVIEW

**Review by EngineeredVirus — CHANGES_REQUESTED** [review]
Need to make adjustments.

File: docs/release-notes/2.0.0.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# Version 2.0.0 (2023-02-)
+
+| Release date |   |
+| ------------ | ------------------------------------------ | 
+| Installation | [Install `pg_stat_monitor`](../install.md) | 
+
+## Release highlights
+
+* `pg_stat_monitor` is Generally Available, enabling you to use it in production environments.
+* Added support of PostgreSQL 15:
```
Comment (LenzGr) [inline docs/release-notes/2.0.0.md:10]:
```suggestion
* Added support for PostgreSQL 15:
```

File: docs/release-notes/2.0.0.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+   | Data type changes | Columns added | Columns renamed | 
+   | ----------------- | ------------- | --------------- |
+   | `userid` - oid <br>  `queryid` - bigint <br> `top_queryid` - bigint <br> | `dbid` <br> `user` <br> `temp_blk_read_time` <br> `temp_blk_write_time` <br> `jit_functions` <br> `jit_generation_time` <br> `jit_inlining_count` <br> `jit_inlining_time` <br> `jit_optimization_count` <br> `jit_optimization_time` <br> `jit_emission_count` <br> `jit_emission_time` | `rows_retrieved` to `rows` <br> `plans_calls` to `calls` | 
+
+* Improved consistency with `pg_stat_statements` - Now `pg_stat_monitor` view has exactly the same column names, columns and data types as the `pg_stat_statements` view for every major version of PostgreSQL from 11 till 15.
```
Comment (LenzGr) [inline docs/release-notes/2.0.0.md:16]:
```suggestion
* Improved consistency with `pg_stat_statements` - The `pg_stat_monitor` view now has the same column names, columns and data types as the `pg_stat_statements` view for every major version of PostgreSQL from 11 to 15. 
```

File: docs/release-notes/2.0.0.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+* Histogram output now allows users to see the whole data range, including the queries with the execution time lower than the minimum time, and exceeding the maximum time specified by the user. 
+* Now the query text is stored in memory, improving the memory buffer management and preventing its overflow
+* CPU and user time are now captured for utility statements, helping you understand the resource consumption for these statements execution. 
+* [PMM](https://www.percona.com/doc/percona-monitoring-and-management/2.x/index.html) integration improvements:
+    * An indication of buckets as "done" vs "current" eliminates the need for the tool to evaluate the bucket status and facilitates the accurate data display 
+    * The generation of a unique ID of a query, independent from PostgreSQL versions, database, users or schemas, improves observability of query planning, execution and performance
+
+### Incompatibility with previous releases:
+
+* The `pg_stat_monitor_settings` is deprecated and removed. To retrieve the configuration parameters, use `pg_settings`.
```
Comment (LenzGr) [inline docs/release-notes/2.0.0.md:26]: The `pg_stat_monitor_settings` what`? Variable?

Comment (nastena1606) [reply docs/release-notes/2.0.0.md:26]: Fixed

File: docs/release-notes/2.0.0.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+* Changes to `pg_stat_monitor` view for all PostgreSQL versions:
+
+    * `bucket_start_time` has now the data type as timestamp with time zone
```
[the remark this answers is not in the document]
Comment (nastena1606) [reply docs/release-notes/2.0.0.md:29]: Added to the table

File: docs/release-notes/2.0.0.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
+# Version 2.0.0 (2023-03-2023)
+
+| Release date |  March 20, 2023 |
+| ------------ | ------------------------------------------ | 
+| Installation | [Install `pg_stat_monitor`](../install.md) | 
+
+## Release highlights
+
+* `pg_stat_monitor` is Generally Available with [Percona Distribution for PostgreSQL](https://www.percona.com/software/postgresql-distribution), enabling you to use it in production environments. Its improved internal architecture leads to fewer lock acquisitions, and therefore an improved performance by approximately 20% when tested using `pgbench`.
```
[the remark this answers is not in the document]
Comment (nastena1606) [reply docs/release-notes/2.0.0.md]: Added 2.0.0 version

**Review by capri1989 — APPROVED** [review]

**Review by Naeem-Akhter — APPROVED** [review]
LGTM!

**Pull request merged**